src/components/Section.js

```diff
@@ -1,12 +1,11 @@
 export default class Section {
-  constructor({ data, renderer }, containerSelector) {
-    this._items = data;
+  constructor({ renderer }, containerSelector) {
     this._renderer = renderer;
     this._container = document.querySelector(containerSelector);
   }
 
   renderItems() {
-    this._items.forEach((item) => this._renderer(item));
+    this._renderer();
```

Comment:
`renderItems` should render the array using `this._container` inside it (or `addItem`)